src/main/java/training/supportbank/Account.java

```diff
@@ -0,0 +1,58 @@
+package training.supportbank;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Account {
+
+    private final String name;
+    private List<Transaction> transactions;

```

Comment:
This can be made `final` as well because the instance variable is never reassigned (even though its values are updated).